.buildkite/run_functional_test.sh

```diff
@@ -28,18 +28,8 @@ export PIP=$VENV_ROOT/bin/pip
 
 $PIP install py-spy
 
-# If we run on buildkite, we connect to docker so we can pull private images
-# !!! WARNING be cautious about the following lines to avoid leaking the secrets in the CI logs
-set +x  # Do not remove so we don't leak passwords
-if [ -v BUILDKITE ]; then
-  echo "Connecting to Vault"
-  VAULT_ADDR=${VAULT_ADDR:-https://vault-ci-prod.elastic.dev}
-  VAULT_USER="docker-swiftypeadmin"
-  echo "Fetching Docker credentials for '$VAULT_USER' from Vault..."
-  DOCKER_USER=$(vault read -address "${VAULT_ADDR}" -field user_20230609 secret/ci/elastic-connectors/${VAULT_USER})
-  DOCKER_PASSWORD=$(vault read -address "${VAULT_ADDR}" -field secret_20230609 secret/ci/elastic-connectors/${VAULT_USER})
-  echo "Done!"
```

Comment:
I'm not sure that our ftests ever needed docker creds. These env vars don't seem to be used anywhere in our ftests.